tests/ValueObjects/ApiToken.php

```diff
@@ -2,10 +2,10 @@
 
 // Generate test for API token value object...
```

Comment:
@filips-dev Can you adjust this comment and adjust the file name of this test?

Reply:
Resolved